recipes/spyder-line-profiler/run_test.py

```diff
@@ -0,0 +1,9 @@
+"""
+Test whether spyder_line_profiler is installed
+
+The test is only whether the module can be found. It does not attempt
+to import the module because this needs an X server.
+"""
+
+import imp
+imp.find_module('spyder_line_profiler')
```

Comment:
We are you using an extra test file instead of adding the `test`/`import`section in the `meta.yaml`?